### PR TITLE
Add Routes, Controllers and Corresponding Views

### PR DESCRIPTION
**Pull request by martinkarugaba**
# Changes made

- [ ] https://
- [ ] https://users/745
- [ ] https://users/745/posts
- [ ] https://users/745/posts/23

### CODE REVIEW

**Comment by zeff96** [comment]
Hello @martinkarugaba

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class UsersController < ApplicationController
```
Comment (zeff96) [inline app/controllers/users_controller.rb:1]: - [ ] Hey there, awesome job on setting up the users controller! 🌟 However, we noticed a small deviation from the project requirements. As per the specifications, the users controller should include both the #index and #show actions. Currently, only the #index action is implemented. 🤔 To align with the project requirements, please ensure to add the corresponding view for the #show action, and don't forget to update the routes to include a route for the users' show page. 🚀 Let's make sure we cover all the bases and keep up the great work! 👏💻

```ruby
# app/controllers/users_controller.rb
class UsersController < ApplicationController
  
  def index; end

  def show; end

end

# config/routes.rb

  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end



```

**Review by zeff96 — CHANGES_REQUESTED** [review]
### Status: CHANGES REQUIRED ♻️
Hi @martinkarugaba , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

congrats

## Highlights
:heavy_check_mark: Linters passing
:heavy_check_mark: Correct GitFlow

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (martinkarugaba) [reply app/controllers/users_controller.rb:1]: Thanks @zeff96 let me look into it

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @martinkarugaba,

#### Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

### Congratulations! 🎉 

giphy (1)

## (Highlights) Good Points: :+1:
- Your work is well documented. ✔
- The correct workflow is used.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**